pages.fr/common/bg.md

```diff
@@ -0,0 +1,11 @@
+# bg
+
+> Reprend l'exécution de tâches qui ont été suspendues (en utilisant `Ctrl + Z` par exemple) en arrière-plan.
+
```

Comment:
```suggestion
- More information: https://manned.org/bg.

```

Reply:
Should we keep links to a copy of man pages ?  I thought that if a user needed more information he would use `man bg`, so I don't see the point to keep a link to manned.

Reply:
Usually, we use manned as a fallback when there isn't a man page around. This link is used in `bg` English page so it is better to have it here for better alignment between pages.

Reply:
Ok, I understand!